app/controllers/Controller.scala

```diff
@@ -6,23 +6,18 @@ import com.scalableminds.util.mvc.ExtendedController
 import com.scalableminds.util.tools.Fox
 import com.typesafe.scalalogging.LazyLogging
 import models.user.User
-import play.api.i18n.{I18nSupport, Messages, MessagesProvider}
+import play.api.i18n.{Messages, MessagesProvider}
 import play.api.libs.json._
 import play.api.mvc.{InjectedController, Request, Result}
 import security.{UserAwareRequestLogging, WkEnv}
 
 import scala.concurrent.ExecutionContext
 
-trait Controller
-    extends InjectedController
-    with ExtendedController
-    with UserAwareRequestLogging
-    with I18nSupport
-    with LazyLogging {
+trait Controller extends InjectedController with ExtendedController with UserAwareRequestLogging with LazyLogging {
```

Comment:
`I18nSupport` is already part of the `ExtendedController`. Therefore, I removed it here from the dependency list to clean up the code a little